docs/source/admin/index.rst

```diff
@@ -23,38 +23,185 @@ Kubernetes deployment
 If you are expecting a large number of users (>50), consider deploying AiiDAlab on a scalable Kubernetes cluster.
 We provide `instructions and deployment scripts <https://github.com/aiidalab/aiidalab-k8s>`__ for this use case.
 
-Run a separate registry
-=======================
+.. _admin-guide:maintain-app-registry:
 
-By default, the AiiDAlab containers use the central `AiiDAlab application registry <https://aiidalab.github.io/aiidalab-registry/>`_ for installing new apps.
-However, maintaining your own registry is straightforward.
+Maintaining an apps registry
+============================
 
-Serve the new registry:
+By default, AiiDAlab is configured to use the `AiiDAlab application registry <https://aiidalab.github.io/aiidalab-registry/>`_ maintained by the AiiDAlab team for installing new apps.
+However, organizations are encouraged to maintain their own registry with apps suitable for their organization's needs.
```

Comment:
Agreed, we should tune the phrasing here a bit. @ltalirz Feel free to create a PR!

As an aside, another way to avoid fragmentation would also be to allow multiple registries. This could be relatively easily achieved by simply concatenating multiple registry URLs: `https://aiidalab.github.io/aiidalab-registry/api/v1:https://example.com/registry/api/v1`.

Reply:
Right, opened https://github.com/aiidalab/aiidalab/pull/234

As for multiple registries, let's think about it when we have a use case